superset/models/helpers.py

```diff
@@ -1173,7 +1173,13 @@ def handle_single_value(value: Optional[FilterValue]) -> Optional[FilterValue]:
             if isinstance(value, str):
                 value = value.strip("\t\n")
 
-                if target_generic_type == utils.GenericDataType.NUMERIC:
+                if (
```

Comment:
I'm going to turn a blind eye to the fact that this function is copy-and-pasted in multiple places. Furthermore it's rather scary there's zero unit tests for the `filter_values_handler` method.